release-notes/10.0/preview/preview2/aspnetcore.md

```diff
@@ -15,6 +22,193 @@ ASP.NET Core updates in .NET 10 Preview 2:
+
+```razor
+<QuickGrid @ref="movieGrid" Items="movies">
+    <PropertyColumn Property="@(m => m.Title)" Title="Title">
+        <ColumnOptions>
+            <input type="search" @bind="titleFilter" @bind:after="@(() => movieGrid.CloseColumnOptionsAsync())" placeholder="Filter by title" />
+        </ColumnOptions>
+    </PropertyColumn>
+    <PropertyColumn Property="@(m => m.Genre)" Title="Genre" />
+    <PropertyColumn Property="@(m => m.ReleaseYear)" Title="Release Year" />
+</QuickGrid>
+
+@code {
+    private QuickGrid<Movie>? movieGrid;
+    private string titleFilter = string.Empty;
+    private IQueryable<Movie> movies = new List<Movie> { ... }.AsQueryable();
+    private IQueryable<Movie> filteredMovies => movies.Where(m => m.Title!.Contains(titleFilter));
+}
+```
+
+Thank you [@matthetherington](https://github.com/matthetherington) for this contribution!
+
+## Populate XML doc comments into OpenAPI document
+
+ASP.NET Core OpenAPI document generation wlll now include metadata from XML doc comments on  method, class, and member definitions in the OpenAPI document. You must enable XML doc comments in your project file to use this feature. You can do this by adding the following property to your project file:
+
+```xml
+<PropertyGroup>
+  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+</PropertyGroup>
+```
+
+At build-time, the OpenAPI package will leverage a source generator to discover XML comments in the current app assembly and any project references and emit source code to insert them into the document via an OpenAPI document transformer.
+
+Note that the C# build process does not capture XML doc comments placed on lambda expresions, so to use XML doc comments to add metadata to a minimal API endpoint, you must define the endpoint handler as a method, put the XML doc comments on the method, and then reference that method from the `MapXXX` method.
+
+For example, to use XML doc comments to add metadata to this minimal API endpoint originally defined as a lambda expression:
+
+```csharp
+app.MapGet("/hello", (string name) =>$"Hello, {name}!");
+```
+
+Change the `MapGet` call to reference a method:
+
+```csharp
+app.MapGet("/hello", Hello);
+```
+
+Then define the `Hello` method with XML doc comments:
+
+```csharp
+static partial class Program
+{
+    /// <summary>
+    /// Sends a greeting.
+    /// </summary>
+    /// <remarks>
+    /// Greeting a person by their name.
+    /// </remarks>
+    /// <param name="name">The name of the person to greet.</param>
+    /// <returns>A greeting.</returns>
+    public static string Hello(string name)
+    {
+        return $"Hello, {name}!";
+    }
+}
+```
+
+Here the `Hello` method is added to the `Program` class, but you can add it to any class in your project.
+
+The example above illustrates the `<summary>`, `<remarks>`, and `<param>` XML doc comments.
```

Comment:
```suggestion
The example above illustrates the `<summary>`, `<remarks>`, and `<param>` XML doc comments.

```

Reply:
I think the intent was to have the following sentence be part of the same paragraph.